extensions/binding/org.eclipse.smarthome.binding.ntp.test/src/test/java/org/eclipse/smarthome/binding/ntp/test/NtpOSGiTest.java

```diff
@@ -284,21 +264,19 @@ public void testDateTimeChannelCalendarDefaultTimeZoneUpdate() {
         // Initialize with configuration with no time zone property set.
         initialize(configuration, NtpBindingConstants.CHANNEL_DATE_TIME, ACCEPTED_ITEM_TYPE_DATE_TIME, null, null);
 
-        String timeZoneIdFromItemRegistry = ((DateTimeType) getItemState(ACCEPTED_ITEM_TYPE_DATE_TIME)).getCalendar()
-                .getTimeZone().getID();
-
+        ZoneOffset timeZoneIdFromItemRegistry = ((DateTimeType) getItemState(ACCEPTED_ITEM_TYPE_DATE_TIME))
+                .getZonedDateTime().getOffset();
         assertThat("The dateTime channel calendar was not updated with the right timezone", timeZoneIdFromItemRegistry,
-                is(equalTo(DEFAULT_TIME_ZONE_ID)));
+                is(equalTo(ZoneOffset.of("+03:00"))));
```

Comment:
Here as well - why do you assume every build server sits in EEST?

Reply:
I am aware of that for sure. :)  But in these tests a default time zone is set to EEST during the tests execution, the underlying system's time zone and locale are stored and restored after the execution of all tests in the method **_tearDownClass()_**  You can take a closer look at **_SetUpClass()_** method where it is described with comments.

Reply:
Got it. All good.